server.js

```diff
@@ -2,28 +2,41 @@ const express = require("express");
 const bodyParser = require("body-parser")
 const cors = require("cors")
 const fs = require("fs");
-const path = require("path")
-
-
+const path = require("path");
+const mongoose = require('mongoose');
 const app = express();
-
 const PORT = 3000;
+const Schema = mongoose.Schema;
 
 app.use(cors());
-
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
-
-app.get("/api/films", (req, res) => {
-    fs.readFile(path.join(__dirname, "films.json"), "utf-8", (err, data) => {
-        if (err) res.json({
-            result: []
-        })
-
+ 
+mongoose.connect('mongodb://localhost/Cinema', function (err) {
+ 
+   if (err) throw err;
+ 
+   console.log('Successfully connected');
+ 
+});
+
+const filmScheme = new Schema({
```

Comment:
you should work on project structure, you can't have all code in one file
first thing to consider is moving out db related code